File: 0683/Sesión 09 - refactorización/refactorizado/SoftInvModel/src/main/java/pe/edu/pucp/softinv/model/PeriodosDTO.java
package pe.edu.pucp.softinv.model;

public class PeriodosDTO {

    private Integer anho;
    private Integer mes;
    private String descripcion;

    public PeriodosDTO() {
        this.anho = null;
        this.mes = null;
        this.descripcion = null;
    }

    public PeriodosDTO(Integer anho, Integer mes, String descripcion) {
        this.anho = anho;
        this.mes = mes;
        this.descripcion = descripcion;
    }

    /**
     * @return the anho
     */
    public Integer getAnho() {
        return anho;
    }

    /**
     * @param anho the anho to set
     */
    public void setAnho(Integer anho) {
        this.anho = anho;
    }

    /**
     * @return the mes
     */
    public Integer getMes() {
        return mes;
    }

    /**
     * @param mes the mes to set
     */
    public void setMes(Integer mes) {
        this.mes = mes;
    }

    /**
     * @return the descripcion
     */
    public String getDescripcion() {
        return descripcion;
    }

    /**
     * @param descripcion the descripcion to set
     */
    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
}

File: 0682/sesión 13 - MasterPage+CRUD/con CRUD/SoftInv/SoftInvModel/PeriodosDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftInv.Model
{
    public class PeriodosDTO
    {
        private int? anho;
        private int? mes;
        private string descripcion;

        public PeriodosDTO()
        {
            this.Anho = null;
            this.Mes = null;
            this.Descripcion = null;
        }

        public PeriodosDTO(int anho, int mes, string descripcion)
        {
            this.Anho = anho;
            this.Mes = mes;
            this.Descripcion = descripcion;
        }

        public int? Anho { get => anho; set => anho = value; }
        public int? Mes { get => mes; set => mes = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
    }
}
